runelite-api/src/main/java/net/runelite/api/Quest.java

```diff
@@ -179,6 +179,7 @@
 	THE_FREMENNIK_EXILES(718, "The Fremennik Exiles"),
 	SINS_OF_THE_FATHER(1276, "Sins of the Father"),
 	A_PORCINE_OF_INTEREST(1690, "A Porcine of Interest"),
+	GETTING_AHEAD(1691, "Getting Ahead"),
```

Comment:
I think this is `752`

Reply:
Indeed this is 752 from what you advised me and what I saw there https://github.com/RuneStar/cs2-scripts/blob/e0bce7a2bc1e27a2910861a9e18140207587508b/scripts/%5Bproc%2Cquestlist_progress%5D.cs2. Thanks :)